packages/core/factories/src/useProductFactory.ts

```diff
@@ -17,23 +18,17 @@ export function useProductFactory<PRODUCT, PRODUCT_SEARCH_PARAMS>(
   factoryParams: UseProductFactoryParams<PRODUCT, PRODUCT_SEARCH_PARAMS>
 ) {
   return function useProduct(cacheId: string): UseProduct<PRODUCT> {
-    console.info(
-      'SSR Temporarly disbled for product composable https://github.com/DivanteLtd/next/issues/232',
-      cacheId
-    );
-    // const { state, persistedResource } = usePersistedState(id);
-
-    // const products: Ref<ProductVariant[]> = ref(state || []);\
-    const products: Ref<PRODUCT[]> = ref([]);
-    const totalProducts: Ref<number> = ref(0);
+    const { state, saveCache } = useSSR(cacheId);
+    const products: Ref<PRODUCT[]> = ref(state || []);
+    const totalProducts = ref(0);
     const loading = ref(false);
 
-    const search = async (params: PRODUCT_SEARCH_PARAMS) => {
+    const search = async (params: any) => {
```

Comment:
why is that?
we need to use `PRODUCT_SEARCH_PARAMS` here :)
```suggestion
    const search = async (params: PRODUCT_SEARCH_PARAMS) => {
```